FinalProject.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+ <head>
+     <meta charset="utf-8">
+     <title>Final Project</title>
+ </head>
+ <body>
+     <header>
+     <h1>Faiza Siddiqi</h1>
+     <nav>
+        <a href="http://www.firefox.com">One</a>
+        <a href="http://www.facebook.com">Two</a>
+        <a href="http://www.google.com">Three</a>
+        <a href="http://www.github.com/faizasidd">Github</a>
+        <br>
+</nav>
```

Comment:
Nitpick: This closing tag should be indented to match the indentation of the opening `<nav>`